pipeline/workflow-builder.Dockerfile

```diff
@@ -13,7 +13,7 @@ FROM ${BUILDER_IMAGE:-registry.redhat.io/openshift-serverless-1/logic-swf-builde
 # When using nightly:
 # ARG QUARKUS_EXTENSIONS=org.kie:kogito-addons-quarkus-jobs-knative-eventing:999-SNAPSHOT,org.kie:kie-addons-quarkus-persistence-jdbc:999-SNAPSHOT,io.quarkus:quarkus-jdbc-postgresql:3.8.4,io.quarkus:quarkus-agroal:3.8.4,org.kie:kie-addons-quarkus-monitoring-prometheus:999-SNAPSHOT,org.kie:kie-addons-quarkus-monitoring-sonataflow:999-SNAPSHOT
 # When using prod:
-ARG QUARKUS_EXTENSIONS=org.kie:kogito-addons-quarkus-jobs-knative-eventing:9.101.0.redhat-00007,org.kie:kie-addons-quarkus-persistence-jdbc:9.101.0.redhat-00007,io.quarkus:quarkus-jdbc-postgresql:3.8.4.redhat-00002,io.quarkus:quarkus-agroal:3.8.4.redhat-00002
+ARG QUARKUS_EXTENSIONS=org.kie:kogito-addons-quarkus-jobs-knative-eventing:9.102.0.redhat-00004,org.kie:kie-addons-quarkus-persistence-jdbc:9.102.0.redhat-00004,io.quarkus:quarkus-jdbc-postgresql:3.8.6.redhat-00004,io.quarkus:quarkus-agroal:3.8.6.redhat-00004
```

Comment:
will the monitoring jars be added later?

Reply:
They are included by default, otherwise we would not had to removed them manually in the dockerfile: https://github.com/rhdhorchestrator/serverless-workflows/pull/512/files#diff-0c21847f7c6713cdbf85fdc53c9414ccbd57cd13fa7b2370ecc8001055b9ec95L37-L39